.github/workflows/k8s_e2e.yml

```diff
@@ -165,7 +165,7 @@ jobs:
           KUBERNETES_VERSION: ${{ matrix.kubernetes_version.version }}
           MINIKUBE_VERSION: ${{ matrix.minikube_version }}
           CONTAINER_RUNTIME: ${{ matrix.container_runtime }}
-      - run: make slim-builds
+          CARGO_INCREMENTAL: 0
```

Comment:
I think this should be moved up to the job level otherwise it only applies to the minikube setup step.

Reply:
Moved it to the right env section...